nnunet/utilities/shutil_sol.py

```diff
@@ -33,5 +33,6 @@ def copytree(src, dst, ignore=None):
 
         if len(non_permission_errors) > 0:
             raise shutil.Error(non_permission_errors)
-
+    except FileExistsError:
```

Comment:
What's the reason for catching these errors? Shouldn't they be caught by the user if they want to ignore them?

Reply:
This does not  happen when starting a single job, but if you are deploying an interactive session where you alter code, and you pip un/install, then you will receive this message of /home/user/{task} data already being there and the code stops. The only way to not receive the error is to remove the copied files and copy them again. This is 10 minutes unnecessarily wasted.

Reply:
Okay - but this should then be accounted for in the place where this function (copytree) is used.